base/server/upgrade/10.10.0/06-AddIPv6AJPAdapter.py

```diff
@@ -0,0 +1,58 @@
+# Authors:
```

Comment:
@edewata  -- do we want this part of this file or part of the server migration? Just wondering if no AJP -> AJP is a path we support? Or would they already be on their own at that point?

Reply:
@edewata OK, per IRC discussion this will now avoid adding a duplicate IPv6 in case one was added manually. :)